geth/log/log.go

```diff
@@ -78,11 +81,13 @@ func SetLevel(level string) {
 // SetLogFile configures logger to write output into file.
 // This call preserves current logging level.
 func SetLogFile(filename string) error {
-	handler, err := log.FileHandler(filename, log.TerminalFormat(false))
-	if err != nil {
-		return err
+	logger.ljWriter = &lumberjack.Logger{
+		Filename:   filename,
+		MaxBackups: 3,  // number of backups
```

Comment:
Minor: maybe we could describe it as `// number of backup files` to denote that `backup = file` in this case. It may not be obvious.